pkgs/misc/tmux-plugins/default.nix

```
@@ -556,6 +556,11 @@ in rec {
     };
   };
 
+  tmux-thumbs = pkgs.callPackage ./tmux-thumbs {
+    inherit lib mkTmuxPlugin fetchFromGitHub;
+    inherit (pkgs.thumbs);
```

```suggestion
    inherit mkTmuxPlugin;
```
There is no need to explicitly supply these inputs as they would be auto filled by callPackage